showcase-quarkus-eventsourcing/reflection-config.json

```diff
@@ -342,6 +342,12 @@
 		"allDeclaredMethods": true,
 		"allDeclaredConstructors": true
 	},
+	{
+		"name": "io.github.joht.showcase.quarkuseventsourcing.message.query.nickname.NicknameDetails",
+		"allDeclaredFields": true,
+		"allDeclaredMethods": true,
+		"allDeclaredConstructors": true
+	},
```

Comment:
This is the main part of the fix. NicknameDetails get serialized using JSON-B directly.
Since JSON-B extracts the information to build the JSON-Structure from the class via reflection,
quarkus (or graalvm) needs to know, that this class needs to be prepared to be called via reflection.